example/symfony28/app/SymfonyRequirements.php

```diff
@@ -257,7 +257,7 @@ public function addPhpIniRecommendation($cfgName, $evaluation, $approveCfgAbsenc
      *
      * @param RequirementCollection $collection A RequirementCollection instance
      */
-    public function addCollection(RequirementCollection $collection)
```

Comment:
Why has this change been made? Surely these were autogenerated by the frameworks?

Reply:
I think this is a StyleCI change because the argument type is the same as the class I assume.  I don't know why it's never happened before, but can this feature to make it more readable.

Reply:
Yes, I can confirm this change was made by StyleCI. I imagine Symfony will update it's generator to use that standard too soon, since it's code style automation tool, the same one that's ultimately used by StyleCI, enforces that change now.